src/scss/themes/_live.scss

```diff
@@ -15,19 +15,22 @@
         color: white;
     }
 
-    .o-teaser__tag:hover {
-        color: oColorsMix(oColorsGetPaletteColor('crimson'), oColorsGetPaletteColor('white'), 30) !important;
-    }
-
     .o-teaser--small.o-teaser--live,
     .o-teaser--hero.o-teaser--live,
-    .o-teaser--hero.o-teaser--live,
     .o-teaser--large.o-teaser--live,
     .o-teaser--top-story.o-teaser--landscape.o-teaser--live {
         * {
             color: oColorsGetPaletteColor('white');
         }
-        *:hover {
+        // *:hover {
+        //     color: oColorsMix(oColorsGetPaletteColor('crimson'), oColorsGetPaletteColor('white'), 30);
+        // }
+
+        .o-teaser__heading:hover,
+        .o-teaser__heading:focus,
+        .o-teaser__tag:hover,
+        .o-teaser__tag:focus,
+        .js-teaser-heading-link:hover {
```

Comment:
How come this is `js-teaser`?

Reply:
Somehow `Heavy - large`'s meta doesn't have the class `o-teaser__tag`

![image](https://user-images.githubusercontent.com/7448436/30819668-5a453920-a217-11e7-83ad-fe03649ffeeb.png)

Reply:
@florianherrengt Is that in a demo or on the live site somewhere?

Reply:
@JakeChampion Not that I'm aware of. I usually send ngrok links to people

Reply:
The specification for origami components states that the classes should begin with `o-` - http://origami.ft.com/docs/component-spec/modules/#naming-conventions

Could we change the selector from `.js-teaser` to `.o-teaser` and ask Next FT to update their html?

Reply:
This commit should fix it https://github.com/Financial-Times/o-teaser/pull/76/commits/1742a69967d3a79eadc2a186a58101bfa83a4541 🙂